src/frontend/templates/pages/Playlists.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useUser } from '../../contexts/userContext';
+import { removePlaylist } from '../../services/playlistService';
+import { toastError, toastSuccess } from '../../services/toastService';
+import { Video } from '../components';
+
+export const Playlists = () => {
+
+    const { userState , userDispatcher } = useUser();
+
+    const RP = async (myToken, playlistDetails) => {
```

Comment:
Be more descriptive in for this variable. RP doesn't seem to be meaningful

Reply:
Hi om, thanks for this ,  i have made changes and given meaningful names to functions